packages/experimental/gem-globe/src/hooks/useTour.ts

```diff
@@ -60,37 +61,31 @@ const startTour = async (controller: GlobeController, features: Features) => {
 
     await d3
       .transition()
-      .duration(Math.max(1_000, distance * 2_000))
+      .duration(Math.max(2_000, distance * 2_000))
       .tween('render', () => (t) => {
-        projection.rotate(iv(t));
-        setRotation(projection.rotate());
+        const t1 = Math.max(0, Math.min(1, t * 2 - 1));
+        const t2 = Math.min(1, t * 2);
```

Comment:
What is `gem-globe` and why are changes to it part of a PR named “Theme plugin tweaks”? Can these concerns be separated?

Reply:
yes could be separated, but fairly minor and ran into problems with them trying to debug style issues here with this. gem is the SVG/canvas lib; globe is the canvas 3d map.